program/src/instructions/collect.rs

```diff
@@ -50,6 +53,7 @@ pub fn process_collect<'info>(
 
         let lamports = account
             .lamports()
+            // TODO: is there a world wher esomeone writes data to one of these acc? I guess not right coz it's a pda.
```

Comment:
The only thing I can think of is if we wanted to store fee data in these accounts so programs can read from them instead of hard-coded constants, but requires program refactoring anyway.